### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by iamricard** [thesayyn/protoc-gen-ts#126] [Q&A] [answered]
Hi!

I know y'all are working hard towards 1.0, but I was wondering if you had any plans to cut a release from `main` before 1.0. We'd love to move over to these rules but our setup struggles with 0.8.1.

**Comment by thesayyn** [comment, marked as answer]
Hi, yeah absolutely. we cut releases frequently. i will cut another one soon.

**Reply by iamricard** [reply]
awesome! looking forward to it 😄

**Reply by thesayyn** [reply]
we've cut 0.8.2. lmk if it works for you.
